example/src/Utilities/APIClient.js

```diff
@@ -45,6 +45,24 @@ class ApiClient {
     return ApiClient.makeRequest(ENVIRONMENT.url + 'paymentMethods', body);
   };
 
+  static removeStoredCard = async (id, configuration) => {
+    let {merchantAccount, shopperReference} = configuration;
+    const url =
+      ENVIRONMENT.url +
+      `storedPaymentMethods/${id}?merchantAccount=${merchantAccount}&shopperReference=${shopperReference}`;
+    const request = new Request(url, {
+      method: 'DELETE',
+      headers: {
+        'Content-Type': 'application/json',
+        'X-API-Key': ENVIRONMENT.apiKey,
+      },
+    });
+    const response = await fetch(request);
+    const pspReference = response.headers.get('pspreference');
+    console.debug(`PSP Reference - ${pspReference}`);
+    return response.status == 204;
```

Comment:
Maybe not to return a `boolean` here because your function's name doesn't indicate it. I think you can return the `response`, and inspect the `response` from the caller.